pool.go

```diff
@@ -402,7 +402,7 @@ func (p *Pool) GetCtx(ctx context.Context, query string, bindings, rebindings ma
 }
 
 // OpenCursorCtx initiates a query on the database, returning a cursor to iterate over the results
-func (p *Pool) OpenCursorCtx(ctx context.Context, query string, bindings, rebindings map[string]string) (cursor Cursor, err error) {
+func (p *Pool) OpenCursorCtx(ctx context.Context, query string, bindings, rebindings map[string]string) (cursor *Cursor, err error) {
```

Comment:
Ensure comment explicitly calls out the kinds of queries and responses that can be handled here, and which functions can then be used for responses

Reply:
pending?